real-quiz.js

```diff
@@ -0,0 +1,35 @@
+var myTree = {
+  height : document.getElementById("height"),
+  character : document.getElementById("character")
+};
+
+var myButton = document.getElementById("growIt");
+
+function makeTreeGrow () {
```

Comment:
you need to pass myTree into this function. an easy way to do that would be to pull out:
 var heightEntered = myTree.height.value;
 var charEntered = myTree.character.value;

into a separate  function (maybe pull the validation stuff out to and have a validation function).  Once your validation is passed call:
makeTreeGrow(myTree);